contrib/index.rst

```diff
@@ -1,28 +1,116 @@
+.. _c_root:
+
 ==================================
 Python Contributor's Guide (draft)
 ==================================
 
+.. raw:: html
+
+    <script>
+    document.addEventListener('DOMContentLoaded', function() {
+      activateTab(getOS());
+    });
+    </script>
+    <style>
+      table.docutils td { vertical-align: top; }
```

Comment:
Perhaps we should move this to `_static/devguide_overrides.css`?

Reply:
I didn't know about that file.  I think tables should default to top-aligned, but idk if this will affect other things badly as a global setting.